debian/scrapyd.install

```diff
@@ -1,3 +1 @@
 debian/000-default etc/scrapyd/conf.d
-debian/tmp/usr/bin/scrapyd
-debian/tmp/usr/lib
```

Comment:
I'm not familiar with debian packaging.
Can you explain me why these lines are deleted?